src/themis/secure_session_message.c

```diff
@@ -170,8 +169,8 @@ themis_status_t secure_session_unwrap(secure_session_t* session_ctx,
                     message_header[i] = iv[CIPHER_MAX_BLOCK_SIZE + i] ^ 0xff;
             }
 
-            length = ntohl(*((uint32_t *)message_header));
-            seq = ntohl(*((uint32_t *)(message_header + sizeof(uint32_t))));
+            length = be32toh(*((uint32_t *)message_header));
```

Comment:
i wonder what it means

https://github.com/cossacklabs/themis/pull/470/files#diff-9c6d9d70482f9c690d698d6bcf60a739R163

Reply:
it's cast array ` uint8_t message_header[sizeof(uint32_t) + sizeof(uint32_t) + sizeof(uint64_t)]` to uint32_t pointer and then dereference pointer to fetch uint32 value to get length. so it's just work with array[4+4+8] as with uint32 pointer and work with first 4 byte value

Reply:
oh sorry, i was trying to make a link to the line above (not part of this PR)

line 163: /* TODO: change to GCM when fixed */

Reply:
we can create an issue to re-review all our comments and todos ) anyway I will read it more deeply in the future when will port secure session to golang gothemis and maybe will understand what was mean it )

Reply:
eventually we will understand what it means :D